CONTRIBUTING.md

```diff
@@ -1,152 +1,196 @@
+    - [Vue components](#vue-components)
+    - [CSS naming](#css-naming)
+  - [Writing and running tests](#writing-and-running-tests)
+  - [Documentation](#documentation)
+  - [Committing rules](#committing-rules)
+  - [Change log entry](#change-log-entry)
+  - [Opening a pull request](#opening-a-pull-request)
+    - [Getting it reviewed](#getting-it-reviewed)
+    - [Contributing to other pull requests](#contributing-to-other-pull-requests)
+  - [Merging pull requests](#merging-pull-requests)
 
 ## Contributing
 
-There are two ways you can contribute to the development. You can either open
-an Issue or if you have programming abilities open a Pull Request.
+There are two ways for you to contribute. You can either open
+an issue or, if you have programming experience, open a pull request.
+
+## Creating an issue
 
-### Creating an Issue
+If you experience a problem while using the Raiden Light Client or want to request a feature,
+then you should open an issue against the repository.
 
-If you experience a problem while using the Raiden Light Client or want to request a feature
-then you should open an issue against the repository. All issues should
-contain:
+### Request features
 
-**For Feature Requests:**
+If you want to submit a feature request, please use the **User Story** issue template and make sure that it contains the following aspects:
 
-- A description of what you would like to see implemented.
-- An explanation of why you believe this would make a good addition to Raiden Light Client.
+- A description of what you would like to see implemented
+- An explanation of your use case
 
-**For Bugs:**
+### Report bugs
+
+If you have found a bug in the Light Client or the JS SDK, then please use the **Bug** issue template and provide the following infos:
 
 - A short description of the problem.
-- Detailed description of your system, SDK version, environment (e.g. Metamask), Wallet version if you are using the wallet.
-- What was the exact unexpected thing that occured.
-- What you were expecting to happen instead.
+- The steps you need take in order to reproduce the behavior
+- What was the expected result
+- What was the actual result
 
-### Creating a Pull Request
+## Creating a pull request
 
-If you have some coding abilities and would like to contribute to the actual
-codebase of Raiden then you can open a Pull Request(PR) against the repository.
+If you have coding experience and would like to contribute to codebase, then you can open a pull request against the repository.
 
 If you are interested in contributing make sure that there is an issue about it. Express interest, by picking the issue so that core developers know that you are working on the issue.
 
-All PRs should be:
+All pull requests should be:
 
 - Self-contained.
 - As short as possible and address a single issue or even a part of an issue.
-  Consider breaking long PRs into smaller ones.
+  Consider breaking long pull requests into smaller ones.
```

Comment:
Maybe remove, since we rarely do this too

Reply:
@nephix Thanks for pointing out, that we are not fully sticking to this rule. I think overall we are doing quite good and improving continuously.

We should stick to it and also address it in PRs ([example from google](https://github.com/GoogleContainerTools/skaffold/pull/2750#pullrequestreview-283621241))
Makes things much more transparent and faster. I would keep it and stick to the rule :)

See eg. https://medium.com/letgo/the-art-of-the-small-pull-request-303f7ef63901

> The Small PR Manifesto

> Our goal is to get PRs integrated onto the master branch as fast as possible. Code isn’t useful until it gets into master.

>Here’s why:

    1. In order to make PRs easy-to-read and review, we’ll make them small.
    2. In order to avoid conflicts in our PRs, we’ll make them small.
    3. In order to handle a single responsibility in each of our PRs, we’ll make them small.
    4. In order to promote healthy and constructive discussion in our PRs, we’ll make them small.
    5. In order to make deployments small and easy to grasp in our PRs, we’ll make them small.
    6. In order to have fast PRs, we’ll make them small.

There are also [tons of articles](https://www.google.com/search?q=small+pull+requests&client=firefox-b-d&ei=ZcT4XdndNMS11fAPhY--qA8&start=10&sa=N&ved=2ahUKEwiZpK2W0rzmAhXEWhUIHYWHD_UQ8NMDegQIDRBC&biw=1280&bih=607), which show the benefit of having small pull requests.

CC @andrevmatos @kelsos

Reply:
The fact that we don't do it is wrong. Smaller PRs are always easier to comprehend and review. The bigger a PR is the easier it becomes to miss details.

Also, this guide is not only for internal use but also for possible external contributors.
This clearly states intent and requirement and removing it might create wrong assumptions about what is acceptable.

Reply:
Lets say we do it almost never.

And I asked whether we should remove it because it doesn't make sense to write down rules and expect others to follow them while we do what suites us